content/en/docs/concepts/overview/kubernetes-api.md

```diff
@@ -197,14 +198,19 @@ Elimination of resources or fields requires following the
 
 Kubernetes makes a strong commitment to maintain compatibility for official Kubernetes APIs
 once they reach general availability (GA), typically at API version `v1`. Additionally,
-Kubernetes keeps compatibility even for _beta_ API versions wherever feasible:
-if you adopt a beta API you can continue to interact with your cluster using that API,
-even after the feature goes stable.
```

Comment:
Removing this seems like a _significant_ change. I know we've had several KEPs discussing this in more detail, but if a user was just looking at our docs (and history of the project) they could have assumed "beta" provided more stability than recent KEPs have intended to provide. Do we need to broadcast these changes more widely?

Reply:
The deprecation policy peer to this explicitly states beta APIs can go away and at what cadence. That, combined with beta APIs *actually* going away at that cadence for the last few years I think is communication enough